documentation/modules/auxiliary/gather/nis_bootparamd_domain.md

```diff
@@ -0,0 +1,98 @@
+## Intro
+
+From the `bootparamd(8)` man page:
+
+> bootparamd is a server process that provides information to diskless clients necessary for booting. It consults the /etc/bootparams file to find the information it needs.
+
+The module documented within will allow a tester to disclose the NIS
+domain name from a server running `bootparamd`. After knowing the domain
+name, the tester can follow up with `auxiliary/gather/nis_ypserv_map` to
+dump a map from an NIS server (running as `ypserv`).
+
+## Setup
+
+Set up NIS as per <https://help.ubuntu.com/community/SettingUpNISHowTo>.
+If the link is down, you can find it via the Wayback Machine.
+
+After that is done, install `bootparamd` however your OS provides it.
+
+Make sure you add a client to the `bootparams` file, which is usually at
+`/etc/bootparams`. The client should be added to `/etc/hosts` if it
+isn't already resolvable.
```

Comment:
It might be worth providing an example `/etc/bootparams` file here.

```
clientname root=nfsserver:/export/clientname/root
```

Reply:
Is `bootparams(5)` not enough? ;)

Reply:
Either way. It would be nice to have a minimal example.

The module documentation makes reference to the `bootparamd` man page, and a link for setting up `bootparamd` on Ubuntu, but neither contain a reference to the bootparams format.

Reply:
I'll definitely add it. Having to find the right man page is unfair.